apps/taiga-lumbermill/src/components/navigation/navigation.component.less

```diff
@@ -1,3 +1,7 @@
 .aside-bar {
     height: calc(100vh - 3rem);
 }
+
+main[tuiNavigationMain] > * {
```

Comment:
seems like problem with router-outlet tag, let's apply display non only for router-outlet for now